core/integration/directory_test.go

```diff
@@ -883,6 +884,114 @@ CMD cat /secret
 		require.NoError(t, err)
 		require.Contains(t, stdout, "***")
 	})
+
+	t.Run("with no .dockerignore", func(ctx context.Context, t *testctx.T) {
+		src := contextDir.
+			WithNewFile("foo.txt", "content in foo file").
+			WithNewFile("Dockerfile",
+				`FROM golang:1.18.2-alpine
+		WORKDIR /src
+		COPY . .
+		`)
+
+		output, err := src.DockerBuild().Directory("/src").File("foo.txt").Contents(ctx)
+		require.NoError(t, err)
+		require.Contains(t, output, "content in foo file")
+	})
+
+	t.Run("use .dockerignore file by default", func(ctx context.Context, t *testctx.T) {
```

Comment:
Fun little edge case lol. https://github.com/dagger/buildkit/blob/f7cf40f994b0ef850330f403e478258c5eab3766/frontend/dockerfile/dockerfile_test.go#L498-L498

`Dockerfile.dockerignore` is valid as well :tada: